src/main/scala/com/cognite/sdk/scala/v1/dataModelProperties.scala

```diff
@@ -14,3 +16,40 @@ final case class Float64Property(value: Double) extends PropertyTypePrimitive
 final case class StringProperty(value: String) extends PropertyTypePrimitive
 
 final case class ArrayProperty[+A <: PropertyTypePrimitive](values: Vector[A]) extends PropertyType
+
+final case class TimeStampProperty(value: ZonedDateTime) extends PropertyType
```

Comment:
Nice, did you check this with the DMS team? What did they say about timezones etc? To my understanding this will get the timezone from the callsite?
```
public static ZonedDateTime now() {
        return now(Clock.systemDefaultZone());
    }
```

Reply:
Actually the type's name timestamp is confused 😅 
According to Erlend it's date time with time zoned aware, so it should be named ZonedDateTime or smthing meaningful than that, and DMS just forward it to pg3, it should be a string with a timestamp format that Postgres recognises. These are stored as UTC and emitted back as UTC. They might need to add the ability to specify a timezone so the api can return the date time matching timezone hehe
Slack ref 👉 https://cognitedata.slack.com/archives/C031G8Y19HP/p1647954609544699?thread_ts=1647950319.267509&cid=C031G8Y19HP

Reply:
Ok, thanks for explaining!